distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogSearchUi.java

```diff
@@ -335,6 +335,21 @@ public void testQueriesWithSpecificSources() {
     assertThat(body.get("src"), is(sources));
   }
 
+  @Test
+  public void testWorkspaceWithQueries() {
```

Comment:
This test should be renamed to be more specific to what is being tested

Reply:
Renamed to `testCreateWorkspaceQueries`